core/scss/components/sidebar/sidebar.scss

```diff
@@ -49,6 +52,15 @@
     &--minimized {
       width: rem-calc(68);
       min-width: 0;
+
+      .md-avatar__img {
```

Comment:
Let's discuss this - should this be a class applied to avatar like sidebar__footer--avatar or target avatar? If targeting avatar is the best solution, then let's use a variable here.

Reply:
its changed to md-sidebar__avatar